pkg/volume/util/ephemeral/ephemeral.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
It looks like there's a push to put any scheduler dependencies into component-helpers: https://github.com/kubernetes/kubernetes/pull/103062

Reply:
Then let's create `k8s.io/component-helpers/storage/ephemeral`.